manifests/emissary/emissary-emissaryns.yaml.in

```diff
@@ -567,31 +566,114 @@ kind: ClusterRole
 metadata:
   labels:
     app.kubernetes.io/instance: emissary-ingress
-    app.kubernetes.io/managed-by: getambassador.io
-    app.kubernetes.io/name: emissary-ingress-agent
-    app.kubernetes.io/part-of: emissary-ingress
-    product: aes
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: agent
+    app.kubernetes.io/version: 1.0.16
+    helm.sh/chart: agent-1.0.16
     rbac.getambassador.io/role-group: emissary-ingress-agent
-  name: emissary-ingress-agent-configmaps
+  name: emissary-ingress-agent-rollouts
 rules:
 - apiGroups:
-  - ""
+  - argoproj.io
   resources:
-  - configmaps
+  - rollouts
+  - rollouts/status
   verbs:
   - get
   - list
   - watch
+  - patch
+---
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.getambassador.io/role-group: emissary-ingress-agent
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/instance: emissary-ingress
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: agent
+    app.kubernetes.io/version: 1.0.16
+    helm.sh/chart: agent-1.0.16
+  name: emissary-ingress-agent
+rules: []
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app.kubernetes.io/instance: emissary-ingress
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: agent
+    app.kubernetes.io/version: 1.0.16
+    helm.sh/chart: agent-1.0.16
+  name: emissary-ingress-agent
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: emissary-ingress-agent
+subjects:
+- kind: ServiceAccount
+  name: emissary-ingress-agent
+  namespace: emissary
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/instance: emissary-ingress
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: agent
+    app.kubernetes.io/version: 1.0.16
+    helm.sh/chart: agent-1.0.16
+  name: emissary-ingress-agent
+  namespace: emissary
+spec:
+  replicas: null
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: emissary-ingress
+      app.kubernetes.io/name: agent
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/instance: emissary-ingress
+        app.kubernetes.io/name: agent
+    spec:
+      containers:
+      - env:
+        - name: LOG_LEVEL
+          value: info
+        - name: AGENT_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: AGENT_CONFIG_RESOURCE_NAME
+          value: emissary-ingress-agent-cloud-token
+        - name: RPC_CONNECTION_ADDRESS
+          value: null
```

Comment:
I don't think this should be `null`, we probably need to set a default in the values.yaml?

Reply:
Thanks, updated the fault.